src/wasm/wasm-binary.cpp

```diff
@@ -2282,6 +2328,50 @@ bool WasmBinaryBuilder::maybeVisitStore(Expression*& out, uint8_t code, bool isA
   return true;
 }
 
+
+bool WasmBinaryBuilder::maybeVisitAtomicRMW(Expression*& out, uint8_t code) {
+  if (code < BinaryConsts::I32AtomicRMWAdd || code > BinaryConsts::I64AtomicRMWXchg32U) return false;
```

Comment:
These bounds look arbitrary without referencing BinaryConsts' definition (and/or the threads proposal).

Would it be worth adding at the end of BinaryConsts' definition something like
```c++
AtomicRMW_BEGIN = I32AtomicRMWAdd,
AtomicRMW_END = I64AtomicRMWXchg32U,
```